tauri-app/src/lib/components/charts/APYTimeFilters.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import ButtonTab from '$lib/components/ButtonTab.svelte';
+  import { ButtonGroup } from 'flowbite-svelte';
+
+  const TIME_DELTA_1_MONTH = 60 * 60 * 24 * 30;
+  const TIME_DELTA_1_YEAR = TIME_DELTA_1_MONTH * 12;
+
+  let now = Math.floor(new Date().getTime() / 1000);
```

Comment:
do we not have a fn for this?

Reply:
no, fn no, but can move to the same place for const TIME literals